cbits/unescape_string.c

```diff
@@ -0,0 +1,150 @@
+#include <string.h>
```

Comment:
// Copyright (c) 2008-2010 Bjoern Hoehrmann <bjoern@hoehrmann.de>
// See http://bjoern.hoehrmann.de/utf-8/decoder/dfa/ for details.
Maybe we should keep the original (c)?